config/default.yaml

```diff
@@ -69,4 +69,4 @@ name_services:
 payments:
   token_address: "0xbf9be54Df2001E6Bd044cED0E508d936A9d38b1D"
 rpc:
-  send_transaction_timeout_secs: 120
+  send_transaction_timeout_secs: 300
```

Comment:
Since the confirmation window presents more information now, including links to storage, I think we should give users more time to review it. Hence, I've increased the timeout to confirm transactions from 2 to 5 minutes.